registry/templates/create_bucket

```diff
@@ -22,5 +23,6 @@ name = '{{ getv "/deis/registry/s3bucket" }}'
 name = '{{ getv "/deis/registry/bucketName" }}'
 {{ end }}
 
-if conn.lookup(name) is None:
+if not conn.lookup(name):
```

Comment:
cf12b888ffc180144e13c455c9d4de8522f942af just changed this from `not` to `is None` - Any reason to revert that?